bids/layout/config/bids.json

```diff
@@ -4,13 +4,13 @@
         {
             "name": "subject",
             "pattern": "[/\\\\]sub-([a-zA-Z0-9]+)",
-            "directory": "{{root}}/{subject}"
+            "directory": "{{root}}{subject}"
```

Comment:
Perhaps it would make sense for the `[/\\\]` be non-capturing? (Non-)Problem for another day, but I'd forget to mention it otherwise.

Reply:
I don't think that would fix it... the problem isn't about capturing, it's about matching. In grabbit, the `pattern` is [injected into the `directory` template string](https://github.com/grabbles/grabbit/blob/master/grabbit/core.py#L783), which then causes problems when attempting to detect two `/` in a row and only one exists. We could of course make the `[/\\\]` optional, but that defeats the point of having it in the first place. I'm sure there's a more elegant solution, but the above seems to work fine for now, and doesn't require modifications to grabbit.